async/kafka/group/group.go

```diff
@@ -164,10 +164,14 @@ func (h handler) ConsumeClaim(sess sarama.ConsumerGroupSession, claim sarama.Con
 	ctx := sess.Context()
 	for msg := range claim.Messages() {
 		kafka.TopicPartitionOffsetDiffGaugeSet(h.consumer.group, msg.Topic, msg.Partition, claim.HighWaterMarkOffset(), msg.Offset)
+		kafka.MessageStatusCountInc(kafka.MessageReceived, h.consumer.group, msg.Topic)
```

Comment:
Would it make sense to catch the metrics outside the main logic ?
(see similar comment below for the producer part)